internal/app/admin/http/cache.go

```diff
@@ -210,6 +233,16 @@ func printCacheEntries(keys []string, cache cache.ReadOnlyCache, w http.Response
 	}
 }
 
+func clearCacheEntries(keys []string, o *orchestrator.Orchestrator, w http.ResponseWriter) {
+	errors := orchestrator.Orchestrator.ClearCacheEntries(*o, keys)
```

Comment:
can we use `o.ClearCacheEntries(keys)`?

Reply:
The function provides an orchestrator.Orchestrator pointer, so we would still need to dereference it `(*o).ClearCacheEntries(keys)`. Most of the other handlers run orchestrator functions using `orchestrator.Orchestrator.FUNCTION(*o, ...)`, but I can also change them in a different PR.